sig-contributor-experience/charter.md

```diff
@@ -76,14 +70,29 @@ CNCF in many of the noted cases above, contributes funding to our platforms, pro
 - Code for the testing and CI infrastructure - that’s SIG Testing
 - [kubernetes/community]  ownership of folders for KEPs and Design Proposals. Members are to follow those folders owners files and SIG leadership for the specific issue/PR in question.
 - User community management. We hold office hours because contributors are a large portion of the volunteers that run that program.
+  - Slack or YouTube moderation (This should be a new SIG or another group)
 - The contributor experience for repos not included in the Kubernetes associated repositories list found in the [GitHub Management] subproject README.
-- Steering committee election policy updates and maintenance.
+- Steering committee election policy updates and maintenance. (Steering Comittee handles this)
 - We do not create SIGs/WGs but can assist in the various community management needs of their micro communities that would kick off their formation and keep them going.
+  - We do not close down Working Groups, that is the responsibility of the sponsoring SIGs of that working group
 - We are not the [code of conduct committee] and therefore do not control incident management reporting or decisions; however, our moderation guidelines allow us to act swiftly if there is a clear violation of terms of either our code of conduct or one of our supported platforms terms of service. If there is an action that the committee needs to take that involves one of these platforms (example: the removal of someone from GitHub), we will carry that out if none of the committee members have access.
+- SIG Properties 
+  - SIG Calendars and their settings
```

Comment:
we are responsible for the upkeep of the shared calendar and making sure that is working

Reply:
we also advise on settings and best practices in order to display on shared calendar or wherever